### DISCUSSION TITLE
Performance issue?

### DISCUSSION THREAD

**Discussion by Icakis** [metalama/Metalama#147] [Q&A] [answered]
Hello, 
I'm testing usage of metalama in our company's project (free license).
I've created OverrideMethodAspect which contains simple try-catch logic for decorating methods.
The problem i'm hitting that memory usage of Roslyn process became huge - 18GB.
![metaLamaRAM](https://github.com/postsharp/Metalama/assets/7414695/68fa267c-7c70-41d0-9167-4b5fae754e1a)

In addition, the project contains several wcf clients with a lot auto-generated code

So my quetion is something i'm doing wrong or someone has the same issue?

**Comment by gfraiteur** [comment, marked as answer]
There is a known and indeed huge memory leak in Metalama 2023.0. We will publish a fix in 2023.1. In the meantime you should restart VS from times to times.

**Reply by svick** [reply]
This has been fixed in version 2023.1.6-rc.
